### PR TITLE
Make a script for running metronome locally

### PR DESCRIPTION
**Pull request by alenkacz**
I had to do these steps like 300 times in the last 2 weeks so I finally automated them through this script :)

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+ZK_URL="${1:-zk://127.0.0.1:2181/metronome}"
+MESOS_MASTER_URL="${2:-127.0.0.1:5050}"
+PLAY_PORT="${3:-8888}"
```
Comment (zen-dog) [inline run.sh]: why don't we call it `HTTP_PORT`? ;) And it `8888` or `8989` a better default one?

Comment (alenkacz) [reply run.sh]: good point, thanks

**Comment by tobilg** [comment]
Slightly off-topic, there is also a repo for a Docker build for Metronome at https://github.com/mesoshq/metronome which could also be used locally.

File: run.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+sbt universal:packageBin
+ZIP_COUNT=`ls target/universal/metronome*.zip | wc -l`
+if [ ${ZIP_COUNT} -gt 1 ]; then
+    echo "Multiple metronome zip files inside /target/universal. Run 'sbt clean' or remove one manually.".
+    exit 1
+fi
+# unpack the package
+unzip -o -d target/universal -a target/universal/metronome-*.zip
+chmod +x target/universal/metronome/metronome-*/bin/metronome
+LIBPROCESS_IP=127.0.0.1 ./target/universal/metronome/metronome-*/bin/metronome -d -v -Dmetronome.framework.name=metronome-dev -Dmetronome.zk.url=$ZK_URL -Dmetronome.mesos.master.url=$MESOS_MASTER_URL -Dplay.server.http.port=$PLAY_PORT
```
Comment (kensipe) [inline run.sh]: `./target/universal/metronome/metronome-*/bin/metronome` should be `./target/universal/metronome-*/bin/metronome`

File: run.sh (hunk 1 above)
Comment (kensipe) [reply run.sh]: agree with zen-dog and think the default should be 9000 like it is every where else

**Comment by kensipe** [comment]
also think it should go in a `run` or `script` dir... but isn't that big of a deal.

**Pull request merged**